### ISSUE TITLE
Issue: "Show CO" Accessibility

### ISSUE THREAD

**Issue by UOcelot16** [Robosturm/Commander_Wars#1400]
**Description**
"Show CO" accessible even no CO has been selected. Even this is a Screenshot of 0.27.1 it's still in there. How about blocking this button for this condition (greyed out)?

**Screenshots**
![grafik](https://user-images.githubusercontent.com/119428442/211860826-c0b5601d-71cd-4e0e-a7ab-47f696f969dc.png)

**Desktop (please complete the following information):**
 - OS: Win10
 - Version: v0.28.2 (shown ingame as 28.1-main)
 - Active Mods: Vanilla

**Comment by Robosturm** [comment]
Yes and i don't see an issue with that. Stays as is since you can always browse through the co's with next an back
